src/js/controllers/TeamImportDialogController.js

```diff
@@ -1,21 +1,24 @@
-define('controllers/TeamImportDialogController',[
+define('controllers/TeamImportDialogController', [
     'services/ng-handshake',
     'angular'
-], function() {
+], function () {
     var moduleName = 'TeamImportDialog';
 
-    return angular.module(moduleName, []).controller('TeamImportDialogController',[
+    return angular.module(moduleName, []).controller('TeamImportDialogController', [
         '$scope', '$handshake',
         function ($scope, $handshake) {
             var defer;
 
             function parseData(data) {
                 //parse raw import, split lines
-                var lines = data?data.split(/[\n\r]/):[];
+                var lines = data ? data.split(/[\n\r]/) : [];
                 if ($scope.importHeader) {
                     lines.shift();
                 }
-                lines = lines.map(function(line) {
+                lines = lines.map(function (line) {
+                    if ($scope.useCustomDelimiter) {
```

Comment:
Unnecessarily complex: just make `$scope.delimiter` the tab character by default, and always split on `$scope.delimiter` then.